arwen/contexts/runtime.go

```diff
@@ -113,6 +113,11 @@ func (context *runtimeContext) PopInstance() {
 }
 
 func (context *runtimeContext) ClearInstanceStack() {
+	if len(context.instanceStack) > 0 {
```

Comment:
no need for this if. going directly over the range is good enough.

Reply:
Fixed.